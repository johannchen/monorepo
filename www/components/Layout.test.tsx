import React from 'react';
import renderer from 'react-test-renderer';
import Layout from './Layout';

test('render layout', () => {
  const component = renderer.create(<Layout />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
