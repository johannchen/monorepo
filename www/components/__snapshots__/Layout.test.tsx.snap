// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render layout 1`] = `
<div>
  <header>
    <nav>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
       |
       
      <a
        href="/about"
        onClick={[Function]}
      >
        About
      </a>
    </nav>
  </header>
  <footer>
    I'm here to stay
  </footer>
</div>
`;
